ui/apps/platform/cypress/fixtures/auth/authProviders-id1.json

```diff
@@ -13,9 +13,7 @@
             "loginUrl": "irrelevant",
             "validated": false,
             "active": false,
-            "traits": {
```

Comment:
Do we have **some** tests that work with "ALLOW_MUTATE"/"ALLOW_MUTATE_FORCE"  mutability modes?